### ISSUE TITLE
Output of yolo

### ISSUE THREAD

**Issue by Nazarko12345** [learnml-today/object-detection-with-pytorch#9]
Hi.Why u get output like  self.fc2 = nn.Linear(4096, g * g * (b * 5+classes)) ?I think than we should have b * (5+c) for every box 4 coors confidence and class ?

**Comment by creotiv** [comment]
Yes, but no) That's how the YoloV1 works. You have additional boxes just to get better coverage of coordinates distribution. 
But of course, it's just a code that you can change, so you can always add more constraints to the network.
There is no right solution, there are just better or worse results, which understood only by testing.

As a solution, having class for each box is a good thing because of an increase accuracy. But from the other side it will need much more data because you basically decrease the amount information for each bbox.
For example, such a scenario used in FasterRCNN network where boxes predicted per class
